src/stories/pages/SearchPage/styles.scss

```diff
@@ -0,0 +1,72 @@
+@import '../../../scss/variables.scss';
```

Comment:
Page components should not have a dedicated 'styles'. Since pages are just boards with a circuit of components, styles for those components should live in components/styles

Reply:
Ah. That makes sense, will make changes per your and Alessia's comments.